Cargo.toml

```diff
@@ -34,3 +34,4 @@ ethers-signers = "0.1.3"
 keccak="0.1.0"
 bn-plus = "0.4.4"
 rust-crypto="^0.2"
+ripemd160="0.9.1"
```

Comment:
I'm guessing this is the crate that `ripemd160port.rs` came from, but I don't see any actual usages of this crate in the code. If there aren't any, this should probably be removed as a dependency from the Cargo file

Reply:
Removed